receiver/postgresqlreceiver/metadata.yaml

```diff
@@ -79,6 +82,22 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [database, table, state]
+  postgresql.index.scans:
+    attributes: []
+    description: The number of index scans on a table.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
```

Comment:
Does this mean that it is not a running count, but a realtime count? It seems more likely that it would be a typical counter, rather than the number of scans happening right now. If it is a snapshot of now, the description could be clearer about that.

Reply:
After doing a tad bit of digging/validating I believe that you're right it fits more of a typical counter. Will change it to a monotonic.